kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -9,331 +7,371 @@
  * })
  */
 define([
-    'bluebird',
     'common/runtime',
     'common/props',
     'common/ui',
     'common/events',
     'common/fsm',
-    'kb_common/html'
-], function(
-    Promise,
-    Runtime,
-    Props,
-    UI,
-    Events,
-    Fsm,
-    html
-) {
+    'kb_common/html',
+], function (Runtime, Props, UI, Events, Fsm, html) {
     'use strict';
 
-    let t = html.tag,
+    const t = html.tag,
         div = t('div'),
         button = t('button'),
         span = t('span'),
         p = t('p'),
-        smallPanelHeight = '300px',
-        largePanelHeight = '600px',
         numLines = 100,
-        panelHeight = smallPanelHeight,
-        // all the states possible, to be fed into the FSM.
-        appStates = [{
+        cssBaseClass = 'kb-log',
+        logContentStandardClass = `${cssBaseClass}__content`,
+        logContentExpandedClass = `${logContentStandardClass}--expanded`;
+
+    let logContentClass = logContentStandardClass;
```

Comment:
Small change here -- rather than saving a pixel height, `panelHeight`, I'm saving the name of the class of the log content.

Prettier changes only up until the next comment.